vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -10,6 +10,9 @@ def textual_group_properties
 
   def textual_group_relationships
     items = %w(container_nodes container_services container_groups)
+    if @ems.class.ems_type.eql? 'openshift'
```

Comment:
Prefer `==` over `.eql?`  However, even so, this should just be `@ems.kind_of?(EmsOpenshift)`